dcos_test_utils/dcos_cli.py

```diff
@@ -93,13 +93,19 @@ def exec_command(self, cmd: str, stdin=None) -> tuple:
 
         log.info('CMD: {!r}'.format(cmd))
 
-        process = subprocess.run(
-            cmd,
-            stdin=stdin,
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
-            env=self.env,
-            check=True)
+        try:
+            process = subprocess.run(
+                cmd,
+                stdin=stdin,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                env=self.env,
```

Comment:
I assume this sets PYTHONIOENCODING to utf8?

Reply:
Yes, that appears to be true.